BrowserKeywords.robot

```diff
@@ -0,0 +1,114 @@
+*** Settings ***
+Library  SeleniumLibrary
+
+*** Variables ***
+
+${url}                          https://moolya.com/
+${browser}                      Chrome
+${Moolya Logo}                  xpath=//img[@class="navbar-brand moolya-logo"]
+${Page Down}                    xpath=//body   
+${End key}                      \ue00f
+
+# Home page Variables
+${Moolya Home page}             xpath= //a[@href="//moolya.com/Home"]

```

Comment:
Use camelcase for naming variables and avoid space

Reply:
Ok noted, Variables are named using camelCase and Extra spaces are removed. Folders are restructured as Resources and Test. Where Keywords and Variables are stored in Resource folder.